server/config/config.go

```diff
@@ -855,6 +856,8 @@ type KeyspaceConfig struct {
 	WaitRegionSplit bool `toml:"wait-region-split" json:"wait-region-split"`
 	// WaitRegionSplitTimeout indicates the max duration to wait region split.
 	WaitRegionSplitTimeout typeutil.Duration `toml:"wait-region-split-timeout" json:"wait-region-split-timeout"`
+	// DisableRawKVRegionSplit indicates whether to skip raw kv region split.
+	DisableRawKVRegionSplit bool `toml:"disable-raw-kv-region-split" json:"disable-raw-kv-region-split,string"`
```

Comment:
how about using `Enable...` as the config name?

Reply:
done, I have change it to simply `SplitRawKVRegion`